cmd/kubelet/app/options/options.go

```diff
@@ -69,6 +69,7 @@ func NewKubeletServer() *KubeletServer {
 		KubeletConfiguration: componentconfig.KubeletConfiguration{
 			Address:                     "0.0.0.0",
 			CAdvisorPort:                4194,
+			CalculateVolumeDiskUsageFrequency: unversioned.Duration{5 * time.Minute},
```

Comment:
The default is way too slow for production use case. Essentially, one won't' know that their container or pod is close to full for nearly 5 minutes (+metrics aggregation and alerting latency)